elasticsearch/azuredeploy.parameters.json

```diff
@@ -3,13 +3,13 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "dnsNameforLBIP": {
-      "value": "changeme"
+      "value": "elasticlbip"
```

Comment:
Please change this value to "#####". Helps the automated bot deploy this and replace this with a random gen'd string